devTools/initializeDeploy.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+// usage: ./devTools/initializeDeploy.js --multnomah
+const db = require('../app/db');
+const readline = require('readline');
+const credentials = require('../credentials');
+const hashPw = require('../app/lib/pass').hashPw;
+
+const ask = (prompt, callback) => {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  rl.question(prompt, (answer) => {
+    callback(answer);
+    rl.close();
+  });
+};
+
+let ORG_NAME, ORG_EMAIL, ORG_TZ, DEPT_NAME, CFA_USERNAME, CFA_FIRST_NAME,
+  CFA_LAST_NAME;
+
+if (process.argv[2] === '--multnomah') {
+  ORG_NAME = 'Multnomah County';
+  ORG_EMAIL = 'tdooner@codeforamerica.org';
+  ORG_TZ = 'America/Los_Angeles';
+  DEPT_NAME = 'Department of Community Justice';
+  CFA_USERNAME = 'tdooner@codeforamerica.org';
+  CFA_FIRST_NAME = 'Tom';
+  CFA_LAST_NAME = 'Dooner';
```

Comment:
Yeah, you're right, though I figured that checking in the username wouldn't be too big a deal as long as the password was separate. Probably worth asking for the username on the command line too.